README.md

```diff
@@ -134,7 +134,7 @@ symmetric difference, blkdiag, induced subgraphs, products (cartesian/scalar)
 
 - **flow operations:** maximum flow
 
-- **matching:** bipartite maximum matching
+- **matching:** Matching functions have been moved to [LG-matching.jl](https://github.com/JuliaGraphs/LG-matching.jl).
```

Comment:
Good catch. It was originally "LG-matching" and of course it didn't work, so I removed the dashes - everywhere but here.